testing/text_index_schema_test.cc

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2025, valkey-search contributors
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD 3-Clause
+ *
+ */
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "src/indexes/text/text_index.h"
+#include "src/indexes/text.h"
+
+namespace valkey_search {
+namespace indexes {
```

Comment:
Maybe have it in the same namespace as the class being tested? (`valkey_search::indexes::text`)